docs/approvals/jira-service-management/index.md

```diff
@@ -0,0 +1,211 @@
+
+The integration will require a user account in JSM. The recommendation is to create a 
+service account specifically for Octopus.
+
+Take note of the password assigned or generated for this user.
+
+### Licensing
+
+For the JSM approval checks to be performed as part of the deployment process, an appropriate Octopus license must be configured in your Octopus instance.
+
+A JSM enabled Octopus license must be requested from Octopus directly, and cannot be managed through the self-service process. To request a license register for the [JSM Early Access Program](https://octopusdeploy.typeform.com/servicenow-eap)
+
+Once you have received your feature-enabled license, you can install it by navigating to **{{Configuration, License}}**.
+
+An enabled license will include a block similar to below:
+
+```xml
+<Features>
+  <JiraServiceManagement>
+    <ValidTo>...</ValidTo>
+  </JiraServiceManagement>
+</Features>
+```
+
+### Configuring JSM connections
+
+:::hint
+The instructions in this section will require an Octopus Deploy Manager or Administrator
+:::
+
+To connect your Octopus Deploy instance to JSM, navigate to **Configuration ➜ Settings ➜ 
+Jira Service Management Integration**.
+
+Check the **Enabled** option
+![JSM Integration Settings page](images/jsm-connections-1.png "width=500")
+
+Click on **ADD CONNECTION** and fill out the details.
+The JSM Base Url should be the root URL and include the protocol e.g. `https://`
+
+![JSM Integration Add Connection](images/jsm-connections-2.png "width=500")
+
+Press **TEST** to ensure that the connection details are working.
+
+Multiple JSM connections are supported, however, each project can only use one JSM 
+connection.
+
+### Configuring Issue Comments
+
+:::hint
+The instructions in this section will require an Octopus Deploy Manager or Administrator
+:::
+
+If enabled, this feature will result in a linked change request having one or more Comments added 
+during the deployment lifecycle which record details about the deployment and its execution status.
+
+To enable this feature navigate to **Configuration ➜ Settings ➜ Jira Service Management 
+Integration**, click the **Customer Comments Enabled** checkbox show below then click **Save**.
+
+![JSM Integration Enable Work Notes](images/jsm-worknotes-settings.png "width=500")
+
+## Configuring deployments
+
+To enforce a deployment to require an approved CR, the **Change Controlled** setting needs to be enabled in **both** the project and the environment it is being deployed to.
+
+### Setting up projects for CR approval
+
+To enable a project to enforce a requirement for an approved CR:
+
+1. Navigate to the project and then **{{Settings,General}}**.
+2. Check the **Jira Service Management Integration ➜ Change Controlled** setting.
+3. Select your JSM connection in the **Jira Service Management Connection** setting and click 
+   **SAVE**.
+
+![JSM Integration Project settings](images/jsm-project-settings.png "width=500")
```

Comment:
Can this image be made narrower, there is a lot of whitespace

Reply:
updated